### ISSUE TITLE
TypeError: Cannot read property 'key' of null

### ISSUE THREAD

**Issue by stefankip** [dawoe/umbraco-redirectsviewer#20]
Sometimes while navigating through Umbraco an error appears in my console:
`TypeError: Cannot read property 'key' of null`
The issue occurs on this line: https://github.com/dawoe/umbraco-redirectsviewer/blob/develop/Source/Our.Umbraco.RedirectsViewer/Web/App_Plugins/RedirectsViewer/controllers/editor.js#L57

**Comment by creativesuspects** [comment]
I was having similar issues where Umbraco would start showing all kinds of errors in console and as a result I would be unable to save document type or content changes. I had to do a hard refresh (CTRL+F5) to make Umbraco work properly again. I noticed these issues were triggered by both the settings dashboard and the create overlay for RedirectsViewer. I was able to fix this issue by adding a `property-alias` attribute to all `<umb-property />` tags. I've submitted a PR #23.
